generated/nidaqmx/_task_modules/channels/ao_channel.py

```diff
@@ -2266,3 +2266,39 @@ def ao_voltage_units(self):
             self._handle, self._name)
         check_for_error(error_code)
 
+    @property
+    def physical_chan_ao_supported_power_up_output_types(self):
+        """
+        :class: list[`nidaqmx.constants.AOPowerUpOutputBehavior`]:
+            Indicates the power up output types supported by the
+            channel.
+        """
+        cfunc = (lib_importer.windll.
+                 DAQmxGetPhysicalChanAOSupportedPowerUpOutputTypes)
```

Comment:
This property does not belong in the `AOChannel` class because it is a physical channel property, not a channel property:
```
int32 __CFUNC DAQmxGetPhysicalChanAOSupportedPowerUpOutputTypes(const char physicalChannel[], int32 *data, uInt32 arraySizeInElements);
```

For the Python API, all of the physical channel properties belong here: https://github.com/ni/nidaqmx-python/blob/master/src/nidaqmx/system/physical_channel.py

See https://www.ni.com/docs/en-US/bundle/ni-daqmx/page/mxcncpts/chans.html for the difference between physical channels and (virtual) channels. 

`AOChannel._name` is a virtual channel name, so it isn't guaranteed to be a valid physical channel name because users may assign a different name when adding the channel to a task.

Reply:
The metadata is incorrect. Compare:

`PhysicalChanAOSupportedOutputTypes`: https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/attributes.py#L22067
`'python_class_name': 'PhysicalChannel',`

`PhysicalChanAOSupportedPowerUpOutputTypes`: https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/attributes.py#L22211
`'python_class_name': 'AOChannel',`

Reply:
Understood. 
- Created a new PR in **grpc-device-scrapigen** which updates `python_class_name` of property `physical_chan_ao_supported_power_up_output_types` to `PhysicalChannel`.  
https://github.com/ni/grpc-device-scrapigen/pull/148
- Used the same metadata (**attributes,py**) to generate **ao_channel.py**.